### ISSUE TITLE
Impersonate dropdown-option in user management not working anymore

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#8401]
The last PR that broke/overwrote the action is from @juliushaertl https://github.com/nextcloud/server/pull/7790

I can work around this with changing: `settings/js/users/users.js`
```js
$userListBody.on('click', '.toggleUserActions', function (event) {
```
to
```js
$userListBody.on('click', '.toggleUserActions > .action', function (event) {
```

Then there is one from @danxuliu https://github.com/nextcloud/server/pull/7668
Which I can "fix" by removing the code block for closing
`$(document).on('mouseup', function () {`

I need help here. can someone fix this properly, so that https://github.com/nextcloud/impersonate/blob/45e1d490c92270a7f9d452cf3d8966b5f0bef32c/js/impersonate.js#L23 is triggered again?
(please change as little as possible in the impersonate.js because I have that revamped locally already)

**Comment by juliushaertl** [comment]
You should be able workaround this in the impersonate app by using `$('#userlist tbody').on('click', '.impersonate', function() {` which will then work since the click event in bound after the `$userListBody.on('click', '.toggleUserActions'` and therefore executed before that.

I'll have a more detailed look tomorrow, to see if we can add a proper fix to the server, so that event propagation will not be stopped there.

**Comment by MorrisJobke** [comment]
Fixed by #8416
